src/EnergyPlus/DataHVACGlobals.cc

```diff
@@ -166,6 +165,10 @@ namespace DataHVACGlobals {
 	int const Coil_CoolingAirToAirVariableSpeed( 28 );
 	int const Coil_HeatingAirToAirVariableSpeed( 29 );
 	int const CoilDX_HeatPumpWaterHeaterVariableSpeed( 30 );
+	int const CoilDX_HeatPumpWaterHeaterWrapped( 31 );
+	
+	int const CoilVRF_FluidTCtrl_Cooling( 32 );
+	int const CoilVRF_FluidTCtrl_Heating( 33 );
```

Comment:
I reordered all these to be 1 - n, so we will have merge conflicts here (you, I, and Bereket). Just be careful if you have to merge DataHVACGlobals since these numbers are now in order.